entproto/cmd/protoc-gen-entgrpc/main.go

```diff
@@ -107,6 +108,7 @@ func (g *serviceGenerator) generate() error {
 			"entIdent":     g.entIdent,
 			"newConverter": g.newConverter,
 			"unquote":      strconv.Unquote,
+			"pluralize":    inflect.Pluralize,
```

Comment:
This function already exists in the templates functions with support for custom rules. 
https://github.com/ent/ent/blob/master/entc/gen/func.go#L40

cc @rotemtam